subcommands/targets/show.go

```diff
@@ -51,6 +52,7 @@ func init() {
 		Args:  cobra.ExactArgs(1),
 	}
 	showCmd.AddCommand(sbomCmd)
+	sbomCmd.Flags().String("download", "", "Download all sboms to given path")
```

Comment:
Should we state somewhere explicitly that it downloads to a given _directory_ (not just any path).